packages/animated/src/withAnimated.tsx

```diff
@@ -37,7 +37,8 @@ const createAnimatedComponent = (Component: any) =>
     const hasInstance: boolean =
       // Function components must use "forwardRef" to avoid being
       // re-rendered on every animation frame.
-      !is.fun(Component) || Component.prototype.isReactComponent
+      !is.fun(Component) ||
+      (Component.prototype && Component.prototype.isReactComponent)
```

Comment:
Let's use `Object.getPrototypeOf(Component).isReactComponent` instead

Reply:
Nice fix. Fixed and force push c4d4cf0.
Buy the way I learned that arrow function do have prototype `.__proto__`, which is `Function`. But it doesn't has `.prototype` since it's not a constructor. Javascript is really weird..

Reply:
Oh, of course. Not sure how I missed that 😆

Reply:
I confused myself, so my suggestion didn't make sense. Please revert the newest commit. 😅

Reply:
Uhhh I didn't think of the original `create-react-class` case 😓 OK revert it

Reply:
Fixed and force push a2d71a7